src/sensors/can/can_manager.hpp

```diff
@@ -0,0 +1,108 @@
+#ifndef CAN_MANAGER_HPP
+#define CAN_MANAGER_HPP
+
+#include "motor.hpp"
+
+#include <map>
+
+/// @brief The maximum number of motors that can be connected to the system. Derrived from the max motors per bus assuming bus 3 only holds non-RM motors
+constexpr uint32_t CAN_MAX_MOTORS = 8u + 8u + 32u;
+
+/// @brief The number of CAN busses that can be used
+constexpr uint32_t CAN_NUM_BUSSES = 3u;
+
+/// @brief CAN bus 1 ID
+constexpr uint32_t CAN_1 = 0u;
+/// @brief CAN bus 2 ID
+constexpr uint32_t CAN_2 = 1u;
+/// @brief CAN bus 3 ID
+constexpr uint32_t CAN_3 = 2u;
+
+// Because of how volatile FlexCAN_T4 objects are, there must only ever exist one instance of this class
+// I'm deleting any copy and assignment constructors that may act to create a new instance
```

Comment:
include this comment somewhere in docs (in CANManager brief and on Flexcan variables)